hack/.quickstartEnv

```diff
@@ -39,4 +50,33 @@ requiredENV() {
     read MGC_ZONE_ROOT_DOMAIN </dev/tty
     echo "export MGC_ZONE_ROOT_DOMAIN for future executions of the script to skip this step"
     fi
-}
\ No newline at end of file
+  else 
+      if [[ -z "${GOOGLE}" ]]; then
+      echo "Enter either credentials created either by CLI or by service account (Please make sure the credentials provided are in JSON format)"
+      read GOOGLE </dev/tty
+        if ! jq -e . <<< "$GOOGLE" >/dev/null 2>&1; then
```

Comment:
would it make sense to do this after the `if [[ -z "${GOOGLE}" ]]; then` has closed, so both the input $GOOGLE or the pre-exported $GOOGLE are both verified?

Reply:
for example:
```
if [[ -z "${GOOGLE}" ]]; then
      echo "Enter either credentials created either by CLI or by service account (Please make sure the credentials provided are in JSON format)"
      read GOOGLE </dev/tty
      echo "export GOOGLE for future executions of the script to skip this step"
fi
if ! jq -e . <<< "$GOOGLE" >/dev/null 2>&1; then
  echo "Credentials provided is not in JSON format"
  exit 1
fi
```